pages/common/xzgrep.md

```diff
@@ -0,0 +1,34 @@
+# xzgrep
+
+> Search possibly compressed files with xz, lzma, gzip, bzip2, lzop, or zstd for regular expressions.
+> All options specified are passed directly to `grep`.
```

Comment:
```suggestion
```
We generally only support 2 lines of page descriptions plus one line for more information link.